### PR TITLE
Section assignment output

### PR DESCRIPTION
**Pull request by LindsayYoung**
[Routing permutations #260](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/260)

## What does this change?
Adds a test script that goes through meaningful permutations and tests the assign secton function

https://docs.google.com/spreadsheets/d/1Ba0617v1wqwf89vYkCQmpNGmi3yI-bfPgXSF8JPqkeQ/edit#gid=0 

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by el-mapache** [comment]
@LindsayYoung Can we add some instructions on running this locally? When I run the tests command, I don't specifically see this output, and I'm not seeing a CSV file saved. Thanks!

**Comment by LindsayYoung** [comment]
Happy to add more docs on this!

**Comment by LindsayYoung** [comment]
Improvements are in! Thanks @el-mapache

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## Contents
```
Comment (el-mapache) [inline README.md]: thank you SO MUCH for adding this!! I was planning on putting in a PR for it! This is great 🎉

**Pull request merged**